Doc/c-api/unicode.rst

```diff
@@ -172,9 +172,9 @@ access internal read-only data of Unicode objects:
                                      Py_UCS4 value)
 
    Write into a canonical representation *data* (as obtained with
```

Comment:
PyUnicode_WRITE() kind parameter type is unsigned int, as PyUnicode_KIND() result in the doc. Can you please update it? Same remark for PyUnicode_READ().

Reply:
Well spotted! Thanks, I've updated the PR.